src/libs/histogram.c

```diff
@@ -1425,6 +1472,11 @@ static void _vectorscope_view_update(dt_lib_histogram_t *d)
       dtgtk_button_set_paint(DTGTK_BUTTON(d->colorspace_button),
                              dtgtk_cairo_paint_jzazbz, CPF_NONE, NULL);
       break;
+    case DT_LIB_HISTOGRAM_VECTORSCOPE_RYB:
+      gtk_widget_set_tooltip_text(d->colorspace_button, _("set view to RYB"));
```

Comment:
This is wrong here, it should be on previous text. The message here describe which mode will be displayed after clicking on it. So the message here should be `"set view to u*v*"` and the message for the previous switch entry must be `"set view to RYB"`.